comms/src/connection/mod.rs

```diff
@@ -20,21 +20,27 @@
 //  WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE
 //  USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+#[macro_use]
+mod macros;
+
 pub mod connection;
 pub mod dealer_proxy;
 pub mod error;
-pub mod message;
 pub mod monitor;
 pub mod net_address;
 pub mod peer_connection;
 pub mod types;
 pub mod zmq;
 
+/// Represents a single message frame.
+pub type Frame = Vec<u8>;
+/// Represents a collection of frames which make up a multipart message.
+pub type FrameSet = Vec<Frame>;
```

Comment:
I don't think ```Frame``` and ```FrameSet``` should be move here, it would be more appropriate to leave it in Messages

Reply:
Ye was thinking the same thing actually :) Will move it to `crate::message`